format.d.ts

```diff
@@ -0,0 +1,21 @@
+type Format = FormatToArray | FormatToString
+
+type  options = {
+    abbreviate?: boolean;
+    appendCountry?: boolean;
+    cleanupPostcode?: boolean;
+    countryCode?: string;
+    fallbackCountryCode?: string;
+}
+type FormatToArray = (input: object, options?: options & {
+    output?: 'array'
+}) => string[];
+
+type FormatToString = (input: object, options?: options & {
```

Comment:
If the function is used without second param, the returning type should be `string`. But now it's ambiguous, so it will be `string | string[]`